tests/source/methods.rs

```diff
@@ -0,0 +1,13 @@
+#![crate_type = "lib"]
+
+// @has /data/relationships/structs/data/0/id 'methods::UnitStruct'
+
+// @has /data/relationships/methods/data/0/id 'method'
+
+// has /included/0/child_methods 'method'
+
+pub struct UnitStruct;
+
+impl UnitStruct {
+    pub fn method_function() {}
```

Comment:
Since to create a method it needs to be attached to some object, I need to identify it somehow, but nothing seems to work. Am I misunderstanding that this is not actually a child of a struct or such? Is there some intermediate type?

Reply:
@nrc, I don't know anything about this, maybe you do?

Reply:
Yeah, basically nothing with impls (and thus methods) is going to work very well yet. We need some work in rustc to add support and that is a bit non-trivial in the general case. We could add support for inherent impls pretty easily, if someone wanted to hack on that I could give instructions.

Reply:
And actually, I thought inherent impls worked, but maybe I was just wrong about that.

Reply:
In which case it might just be best for now if we put tests for what works, mark what doesn't (but still keep the code there), and have the tests written for whatever works. Thoughts @steveklabnik?

Reply:
I'd remove the code for the stuff that doesn't work, and go with a `_ =>`. But yeah, otherwise, let's do that, @projektir . It might also be worth noting this over at https://github.com/steveklabnik/rustdoc/issues/7

Thanks @nrc